tests/foreman/ui/test_host.py

```diff
@@ -8,7 +8,7 @@
 
 :CaseComponent: Hosts
 
-:Assignee: pdragun
+:Assignee: tstrych
```

Comment:
Please don't change the assignee of these tests

Reply:
updated.